trusona-sdk/src/main/java/com/trusona/sdk/Trusona.java

```diff
@@ -17,6 +17,7 @@
 import com.trusona.sdk.http.client.v2.service.IdentityDocumentService;
```

Comment:
There's a lot of whitespace changes in this file which I always annoys me when the diffs are so small. I am using the `GoogleStyle` though which I believe is agreed upon in "How we work" or whatever document holds that. 

<img width="720" alt="Screen Shot 2020-06-26 at 8 54 57 AM" src="https://user-images.githubusercontent.com/49415226/85876541-b77cca00-b78a-11ea-8c8a-8f087bd587f8.png">

Maybe this file just hasn't gotten the treatment yet?